src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/TrustDetailsValidator.java

```diff
@@ -106,16 +130,41 @@ private void validateCeasedDate(OverseasEntitySubmissionDto overseasEntitySubmis
             if (UtilsValidators.isNotNull(trustCeasedDate, qualifiedFieldName, errors, loggingContext)) {
                 DateValidators.isDateInPast(trustCeasedDate, qualifiedFieldName, errors, loggingContext);
                 DateValidators.isCeasedDateOnOrAfterCreationDate(trustCeasedDate, creationDate, qualifiedFieldName, errors, loggingContext);
-            }
+
+                if (trustDataDto.getIndividuals() != null && !trustDataDto.getIndividuals().isEmpty()) {
+                    DateValidators.isCeasedDateOnOrAfterIndividualsDateOfBirth(trustCeasedDate, trustDataDto.getIndividuals(), qualifiedFieldName, errors, loggingContext);
+                }
+           }
         }
     }
 
-    private boolean isTrustStillRequired(OverseasEntitySubmissionDto overseasEntitySubmissionDto, String trustId) {
+    private boolean noBeneficialOwnersArePresent(OverseasEntitySubmissionDto overseasEntitySubmissionDto) {
```

Comment:
I'd rename the noBeneficialOwnersArePresent function as it is checking if they are disassociated, you can still have BO's present but disassociated.

Reply:
Renamed to reflect this